src/Uno.Foundation/Metadata/ApiInformation.cs

```diff
@@ -29,7 +29,10 @@ internal static void RegisterAssembly(Assembly assembly)
 		{
 			lock (_assemblies)
 			{
-				_assemblies.Add(assembly);
+				if (!_assemblies.Contains(assembly))
+				{
+					_assemblies.Add(assembly);
+				}
```

Comment:
Will a `HashSet` perform better?

Reply:
We're adding 3 or 4 values, it's likely not going to change much.